data/t_np3.json

```diff
@@ -0,0 +1,12 @@
+{
+  "node_providers":[
```

Comment:
Do we need all these test files and test data? If we use a set we shouldn't even have to test.

More logic -> more code -> more tests -> worse readability -> 😫 devs